addons/csw/functions/fnc_weaponInit.sqf

```diff
@@ -0,0 +1,26 @@
+/*
+ * Author: Brandon (TCVM)
+ * Initializes weapon to disable weapon disassembling
+ *
+ * Arguments:
+ * 0: Weapon <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [weapon] call ace_csw_fnc_weaponInit
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+params["_weapon"];
+
+if (isNull(configFile >> "CfgVehicles" >> typeof(_weapon) >> QGVAR(options) >> "enabled") || {getNumber(configFile >> "CfgVehicles" >> typeof(_weapon) >> QGVAR(options) >> "enabled") != 1}) exitWith {};
+if (_weapon getVariable[QGVAR(initialized), false]) exitWith {};
+if (!(_weapon getVariable[QGVAR(enableCSW), false])) exitWith {};
```

Comment:
default value on 3den attribute is true, only gets save if changed so default value should match.  
 .e.g. I put down static, check it's attribute is pre-checked, but the var wasn't set in-game